### PR TITLE
Customizing API Visibility for Dev Portal Catalog

### PR DESCRIPTION
**Pull request by nerdydread**
Added details for how a user would dynamically show/hide API's on the developer portal.

### CODE REVIEW

**Review by marksou — APPROVED** [review]
👍

File: tyk-docs/content/tyk-developer-portal/customise/customize-api-visibility.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+
+This flag can also be [set programatically](https://tyk.io/docs/tyk-developer-portal/customise/custom-developer-portal/#updating-a-developer-example-adding-custom-fields).
+
+
+## Modify the Portal Catalogue Template to add Show/Hide Logic
+
+The developer portal is fully customizable via templates. We'll add custom logic to the portal catalogue template (catalogue.html) to show/hide the "Internal API" catalogue based on the value of the "internal" flag for the developer.  
+
+Please see the customised catalogue template ​​here​: https://gist.github.com/nerdydread/1c4f2c909e33591d7c63f805e8e3698d
```
Comment (sedkis) [inline tyk-docs/content/tyk-developer-portal/customise/customize-api-visibility.md]: Can we please host this somewhere official ?
Either hard-coded in the docs, as a collapsible block or somewhere else pleases

**Review by sedkis — CHANGES_REQUESTED** [review]
LGTM besides 1 comment - tested,  very easy to follow!  Thanks @nerdydread

**Review by sedkis — APPROVED** [review]

**Comment by sedkis** [comment]
@marksou  can we. merge please

File: tyk-docs/content/tyk-developer-portal/customise/customize-api-visibility.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+```
+</details>
+
+
+We're now going to overwrite the default catalogue.html template in the 'portal/templates' directory on the Tyk Dashboard instance with the custom one above.
+=======
+Please see the customised catalogue template ​​here​: https://gist.github.com/nerdydread/1c4f2c909e33591d7c63f805e8e3698d  
+
+We're now going to overwrite the default `catalogue.html` template in the 'portal/templates' directory on the Tyk Dashboard instance with the custom one.
+>>>>>>> fd0e9c9915af0453fcbb2f477232a68e632ded5a
```
Comment (buger) [inline tyk-docs/content/tyk-developer-portal/customise/customize-api-visibility.md]: I guess it is Github conflict?

Comment (marksou) [reply tyk-docs/content/tyk-developer-portal/customise/customize-api-visibility.md]: Sorted

**Pull request merged**